### DISCUSSION TITLE
全面理解 ActiveRecord · /activerecord

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#507] [Announcements]
https://draveness.me/activerecord

**Comment by Madao-3** [comment]
非常棒，足够耐心！

**Comment by levi0214** [comment]
这篇文章太牛逼了。请问你博客的图是用什么工具做的？

**Comment by wujunze** [comment]
> 这篇文章太牛逼了。请问你博客的图是用什么工具做的？

文章末尾有说明  `图片使用 Sketch 进行绘制。`

**Comment by draveness** [comment]
> > 在这里不像展开介绍 SQL 语法树的生成过程，因为过程比较复杂，详细分析也没有太大的意义。
> 这里应该是「不想展开」？捉个typo。

已经修复了，感谢支持
